packages/kuma-gui/cypress/support/step_definitions/index.ts

```diff
@@ -1,352 +1,8 @@
-  cy.go(direction)
-})
-
-// assert
-
-Then('the URL is {string}', (expected: string) => {
-  const base = Cypress.config('baseUrl') || Cypress.env().KUMA_BASE_URL
-  cy.url().should('eq', `${base}${expected}`)
-})
-
-Then(/^the URL( doesn't | )contain[s]? "(.*)"$/, (assertion: string, str: string) => {
-  const negative = assertion !== ' '
-  const prefix = negative ? 'not.' : ''
-  cy.url().should(`${prefix}include`, str)
-})
-
-// whilst the response is sent after waitForRequest resolves the response itself
-// has been rendered on the server therefore any mocking done after this step
-// will apply to the following request
-Then(/^the URL "(.*)" was?(n't | not | )requested with$/, (url: string, not: string = '', yaml: string) => {
-  cy.wrap({
-    url,
-    ...YAML.parse(yaml) as {
-      method: string
-      searchParams: Record<string, string>
-      body: Record<string, unknown>
-    },
-  }).then(async (request) => {
-    const found = await client.waitForRequest(request)
-    if (not.trim().length === 0) {
-      expect(found).to.equal(true, `${url} was requested with ...`)
-    } else {
-      expect(found).to.equal(false, `${url} wasn't requested ...`)
-    }
-  })
-})
-
-Then(/^the "(.*)" element[s]? exist[s]? ([0-9]*) time[s]?$/, (selector: string, count: string) => {
-  $(selector).should('have.length', count)
-})
-
-Then(/^the "(.*)" element[s]?( don't | doesn't | )exist[s]?$/, function (selector: string, assertion: string) {
-  const negative = assertion !== ' '
-  const prefix = negative ? 'not.' : ''
-
-  $(selector, {
-    ...timeout(negative),
-  }).should(`${prefix}exist`)
-})
-
-Then(/^the "(.*)" element[s]?( isn't | aren't | is | are )(checked|disabled)$/, (selector: string, assertion: string, booleanAttribute: string) => {
-  const negative = !['is', 'are'].includes(assertion.trim())
-  const prefix = negative ? 'not.' : ''
-
-  $(selector, {
-    ...timeout(negative),
-  }).should(`${prefix}be.${booleanAttribute}`)
-})
-
-Then(/^the "(.*)" element(s)? contain[s]?$/, (selector: string, multiple = '', table: DataTable) => {
-  const rows = table.rows()
-  if (multiple === 's') {
-    $(selector).each((el, i) => {
-      const item = rows[i]
-      if (item) {
-        cy.wrap(el).contains(item[0])
-      }
-    })
-  } else {
-    rows.forEach((item) => {
-      $(selector).contains(item[0])
-    })
-  }
-})
-Then(/^the "(.*)" element( doesn't | )contain[s]? "(.*)"$/, (selector: string, assertion: string, value: string) => {
-  $(selector).then(($el) => {
-    const negative = assertion !== ' '
-    const prefix = negative ? 'not.' : ''
-    const chainer = $el[0].tagName === 'INPUT' ? 'have.value' : 'contain'
-    $(selector).should(`${prefix}${chainer}`, value)
-  })
-})
-
-Then(/^the "(.*)" element is empty$/, (selector: string) => {
-  $(selector).should('be.empty')
-})
-
-Then('the page title contains {string}', function (title: string) {
-  cy.title().should('contain', title)
-})
+const client = useClient()
```

Comment:
Following my very brief look at this yesterday and our very short quick chat, I didn't notice that these are actually still here (I was wondering why we didn't need to use these anymore, but we do still have them)